lib/_http_client.js

```diff
@@ -43,9 +43,17 @@ const errors = require('internal/errors');
 
 const INVALID_PATH_REGEX = /[^\u0021-\u00ff]/;
 
+//TypeError and error constants
```

Comment:
Style: space after `//` and please punctuate the sentence.

Have you run `make test`?  I would expect the linter to complain.

Reply:
@bnoordhuis Thanks for reviewing,

Updated. I tried but I didn't any linting issues.